### PR TITLE
enh(sqf) improve function name highlighting (slightly broader)

### PR DESCRIPTION
**Pull request by JonBons**
I've tweaked the function matching regex for the sqf language so that it can match functions with the `myTag_component_fnc_initVehicle` naming convention which is common with popular mods for arma, CBA, ACE, etc.

### Changes
Changed regex for sqf functions to `\w+_fnc_\w+` which matches the whole word now instead of just one section.

### Checklist
- [x] Updated markup tests,
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

**Comment by JonBons** [comment]
I have not updated the changelog yet as I did not see a Version next or whatever to place it under and I don't want to cause a merge conflict. Where should I place it?

**Comment by joshgoebel** [comment]
Thanks!

The topmost section is the v11 stuff it would go there, at the bottom of that section with all the other grammar improvements.

File: src/languages/sqf.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default function(hljs) {
   // https://community.bistudio.com/wiki/Functions_Library_(Arma_3)#Adding_a_Function
   const FUNCTION = {
     className: 'title',
-    begin: /[a-zA-Z][a-zA-Z0-9]+_fnc_\w*/
+    begin: /\w+_fnc_\w+/
```
[the remark this answers is not in the document]
Comment (JonBons) [reply src/languages/sqf.js]: Yeah, tested and I believe this should be good.

File: CHANGES.md [hunk 2]
```diff
@@ -82,6 +82,7 @@ Grammars:
 - enh(nginx) improving highlighting of some sections [Josh Goebel][]
 - fix(vim) variable names may not be zero length [Josh Goebel][]
 - enh(sqf) Updated keywords to Arma 3 v2.02 (#3084) [R3voA3][]
+- enh(sqf) Refactored function regex to match CBA component func naming scheme (#3181) [JonBons][]
```
Comment (joshgoebel) [inline CHANGES.md:85]: You need to also add the link on line ~125 or so.

Comment (JonBons) [reply CHANGES.md:85]: :+1:

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**